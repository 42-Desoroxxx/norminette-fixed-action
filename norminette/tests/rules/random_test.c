arr = (char **)malloc(sizeof(char *));
