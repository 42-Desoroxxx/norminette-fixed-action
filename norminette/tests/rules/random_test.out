[36mrandom_test.c[0m - [32mIsAssignation[0m In "GlobalScope" from "None" line 1":
		<IDENTIFIER=arr> <SPACE> <ASSIGN> <SPACE> <LPARENTHESIS> <CHAR> <SPACE> <MULT> <MULT> <RPARENTHESIS> <IDENTIFIER=malloc> <LPARENTHESIS> <SIZEOF> <LPARENTHESIS> <CHAR> <SPACE> <MULT> <RPARENTHESIS> <RPARENTHESIS> <SEMI_COLON> <NEWLINE>
random_test.c: OK!
